copier/main.py

```diff
@@ -376,13 +376,21 @@ def jinja_env(self) -> SandboxedEnvironment:
         """
         paths = [str(self.template.local_abspath)]
         loader = FileSystemLoader(paths)
+        default_extensions = [
+            "cookiecutter.extensions.JsonifyExtension",
+            "cookiecutter.extensions.RandomStringExtension",
+            "cookiecutter.extensions.SlugifyExtension",
```

Comment:
The point above makes me think about this one extension...

Cookiecutter is a big dependency. Since you're adding a way to easily include external extensions... Would it be acceptable to you to remove the whole cookiecutter thing?

After all, if you just `pipx inject copier cookiecutter` and add this extension to your template (or maybe [jinja2-slug](https://pypi.org/project/jinja2-slug/) if that's all you want), it should be enough for most cases.

Reply:
You're totally right, I was a bit troubled with adding such a dependency anyway!